lib/FilesHooks.php

```diff
@@ -396,7 +396,10 @@ protected function fileMoving($oldPath, $newPath) {
 		}
 		$accessList = $this->getUserPathsFromPath($parentPath, $parentOwner);
 		$affectedUsers = $accessList['users'];
-		$oldUsers = $this->oldAccessList['users'];
+        if(!isset($this->oldAccessList['users'])) return;
```

Comment:
```suggestion
	if (!isset($this->oldAccessList['users'])) {
		return;
	}
```

Reply:
Is it really needed to return early? Aren't we losing the handling of potential remote users?